src/main/java/com/jfrog/ide/idea/ui/jcef/message/MessagePacker.java

```diff
@@ -0,0 +1,32 @@
+package com.jfrog.ide.idea.ui.jcef.message;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.intellij.ui.jcef.JBCefBrowserBase;
+
+import static com.jfrog.ide.idea.ui.jcef.message.PackedMessage.IDE_SEND_FUNCTION_NAME;
+
+public class MessagePacker implements MessagePipe {
+    JBCefBrowserBase browser;
+
+    public MessagePacker(JBCefBrowserBase browser) {
+        this.browser = browser;
+    }
+
+    public void send(String type, Object data) {
+        try {
+            String raw = MessagePipeSupport.Pack(type, data);
+            this.send(raw);
+        } catch (JsonProcessingException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    private void send(String raw) {
```

Comment:
```suggestion
    private void executeJavaScript(String raw) {
```

Reply:
I actually think `send` is a better name for this method.